css/styles.css

```
@@ -1 +1,40 @@
-/* CSS */
\ No newline at end of file
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+:root {
+--light-grey--: #e4e4e4;
+--pale-grey--:  #f2f2f2;
+--grey--: #808080;
+--sky-blue--: #87cefa;
+}
+
+body {
+    font-family: 'Roboto', Helvetica, Arial sans-serif;
+    margin: 0;
+}
+
+/* Cookie banner*/
```

Please add missing space at the end of comment 😉
```suggestion
/* Cookie banner */
```

---

done